### ISSUE TITLE
vpat 45: edit bibliography dialog and voiceover

### ISSUE THREAD

**Issue by abaevbog** [zotero/zotero#4407]
VPAT 45 indicates that there is not enough clarity how to use the "Edit bibliography" dialog, suggesting to change focus order or to add instructions. I think the focus order is not an issue (as the issue description indicates a misunderstanding of what itemTree and collectionTree are for) but it's probably fair to say that we need some additional instructions on how to use the dialog.

Ideally, we would have a general way to announce arbitrary messages to screen readers to provide users with such information. It's trivial for NVDA and JAWS by using `aria-live` regions but voiceover doesn't handle `aria-live` from, what looks like, XUL windows. It means that such announcements works just fine in the reader but not in the library tab, edit bibliography dialog, or elsewhere.

Having tried to find a workaround for voiceover, I ran into the [bug report](https://bugzilla.mozilla.org/show_bug.cgi?id=1733266) that seems to boil down to exactly the same issue.

So my thinking is that until this is fixed, we can do a similar thing as with QuickFormat dialog and just add `aria-description` properties for all components to indicate what is their purpose. It's not as neat and it will be trickier to put together sentences that will make sense on each component separately but it might be the best solution for now.

VPAT 45 for the record: 

---

Description: In the Zotero Word Plugin's 'Edit Bibliography' modal, screen reader users must explore the dialog to discover that they need to press the Tab key approximately three times to reach the list of editable bibliographies. Upon opening the modal, focus is initially placed on an edit field called "All Fields & Tags." This edit field only filters results in the first list of citations and does not allow for editing. Its purpose in this context is unclear, as it presents citations not yet included in the document. To access the list of cited materials that can be edited, users must press the Tab key three times. After selecting a bibliography to edit using the arrow keys, users need to press the Tab key a few more times to reach the edit field where changes can be made. The focus order in this modal should be revised to initially place focus on the list of editable bibliographies, bypassing the extraneous collection tree view and item list. 

Impact Statement: Users relying on assistive technologies would benefit from clearer instructions on how to operate dialogs like this; the current focus order and lack of guidance leads to confusion and hinders the completion of tasks. 

Expected Result: When choosing to edit a bibliography, focus should be placed on the list of editable bibliographies upon opening the modal, allowing for immediate interaction using arrow keys to select a bibliography for editing. If controlling the focus order or eliminating the extra fields is not feasible, providing textual instructions to guide users to the editable bibliographies using the Tab key would help reduce confusion. 

Actual Result: Focus starts on the "All Fields & Tags" edit field when the modal is opened, requiring the user to press the Tab key three times to access the list of editable bibliographies
